Source/WebGPU/WebGPU/RenderPipeline.mm

```diff
@@ -30,6 +30,14 @@
 #import "BindGroupLayout.h"
 #import "Device.h"
 
+#if USE(APPLE_INTERNAL_SDK)
+#import <Metal/MTLRenderPipeline_Private.h>
+#else
+@interface MTLRenderPipelineDescriptor ()
+- (void)setSampleMask:(NSUInteger)mask;
+@end
+#endif
```

Comment:
Can we not do this?

What would it take to implement without anything private?

Reply:
(Somehow the other browsers have implemented this without anything private)

Reply:
We'd need a WGSL compiler hook that adds code to the fragment shader if mask != 0xffffffff

Reply:
Do we need to add a bug to implement this feature in our WGSL compiler?

Reply:
We should @djg - I just filed one https://bugs.webkit.org/show_bug.cgi?id=252406